### DISCUSSION TITLE
Is it possible to set a timer on the EasyApply bot?

### DISCUSSION THREAD

**Discussion by dmike23** [voidbydefault/EasyApplyBot#50] [Ideas]
Hey. Just wondering if its possible to set a timer so the bot stops after x amount of jobs applied or x amount of time? Tried reading through everything and not seeing anything on this. Is this something you've considered or am I missing it?

Best!

**Comment by voidbydefault** [comment]
Hi @dmike23 ,

Apologies for not being able to respond. Bot only takes random breaks and scrolls (to mimic human behavior) or no more jobs available to apply but not programmed for conditional stops. You can probably refine criteria to set `config.yaml` to apply for jobs posted in 24 hours only.
